### DISCUSSION TITLE
✅ VPS Configuration Problem with two different Passwords

### DISCUSSION THREAD

**Discussion by GreimersStefan** [isontheline/pro.webssh.net#467] [Q&A] [answered]
Dear WebSSH team,

I have a configuration problem with the software on my macOS 11. 6. 1.
My VPS via SSH has the VNC IP and the associated VNC password.
The username is “root. ” This user has an additional password to configure the VPS.

In the screenshot of the software, however, only the user and the password are shown.

The private key goes through WireGuard and has nothing to do with the problem.

Many greetings stefangreimers

![WebSSH - Configuration German Language](https://user-images.githubusercontent.com/95761102/145230877-5c15d433-839c-441b-b2fe-6eb561d10d69.png)

**Comment by isontheline** [comment]
Hello @stefangreimers 

Thank you a lot for using WebSSH 🙏

I have read several times your issue but I don't understand the question behind.

Which problem do you encounter?

**Comment by GreimersStefan** [comment, marked as answer]
Dear SSH-Team,

I had to understood the issue myselves because of I had false informations from It-specials.

Today I know that I it is not neccessary to use your software.

The configuration I have on the MAC is only with WireGuard, a SSH-Client configuration is further not neccessary.

Excuse me for this circumstances.

Kind Regards
Stefan Greimers

**Reply by isontheline** [reply]
You're Welcome Stefan
